src/tribler-gui/tribler_gui/widgets/downloadspage.py

```diff
@@ -162,10 +162,13 @@ def on_received_downloads(self, downloads):
 
         items = []
         for download in downloads["downloads"]:
-            # Update download progress information for torrents in the Channels GUI
-            self.window().core_manager.events_manager.node_info_updated.emit(
-                {"infohash": download["infohash"], "progress": download["progress"]}
-            )
+            # Update download progress information for torrents in the Channels GUI.
+            # We skip updating progress information for ChannelTorrents because otherwise it interferes
+            # with channel processing progress updates
+            if not download["channel_download"]:
```

Comment:
Are you sure that key "channel_download" always presents?

Reply:
Yes. 
https://github.com/Tribler/tribler/blob/e309a3ec29444659d5bbe097af172b7940eb8108/src/tribler-core/tribler_core/modules/libtorrent/restapi/downloads_endpoint.py#L263